### ISSUE TITLE
Outdated statements in 'Managing dependencies' post?

### ISSUE THREAD

**Issue by FelixAkk** [ember-cli/ember-cli.github.io#1]
Hi there, 

Thanks for the great documentation first off all :) has been of great help building with Ember-CLI!

Excuse me if I'm missing something obvious, this is already fixed down the line, or something with my setup. I'm still very new to Ember.

I'm working with Ember 2.1.0 and Ember-CLI 1.13.8 and was implementing dependency management with Funnel, as according to [the documentation here](http://www.ember-cli.com/managing-dependencies/#using-broccoli-funnel). Currently the documentation mentions "closing" your `ember-cli-build.js` with `module.exports = ...` like mentioned [here](https://github.com/ember-cli/ember-cli.github.io/blame/64510d2d676699bb0bab3b8532639d83cc211575/_posts/2013-04-08-managing-dependencies.md#L272).

This while out of the box with a newly generated Ember project my `ember-cli-build.js` closes with `return ...` instead. When adopting the module exports fashion, my Ember build task crashes with

```
Cannot read property 'rebuild' of undefined
TypeError: Cannot read property 'rebuild' of undefined
    at Builder.wrapIfNecessary (/ember-project/node_modules/ember-cli/node_modules/broccoli/lib/builder.js:156:18)
    at readAndReturnNodeFor (/ember-project/node_modules/ember-cli/node_modules/broccoli/lib/builder.js:61:20)
    at /ember-project/node_modules/ember-cli/node_modules/broccoli/lib/builder.js:42:14
    at lib$rsvp$$internal$$tryCatch (/ember-project/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/ember-project/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:505:17)
    at /ember-project/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1001:13
    at lib$rsvp$asap$$flush (/ember-project/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1198:9)
    at process._tickCallback (node.js:355:11)
```

Should the documentation perhaps be using `return ...` as well? The commit adding that is about a year and a few months old, which lead me to think it's perhaps an outdated convention.

**TL;DR: Using the `return app.toTree( ... );` style exit statement for `ember-cli-build.js`, contrary to the documentation, is correct?**

**Comment by leo** [comment]
Thank you, too, for those great words! :blush: I'll [fix this](http://ember-cli.github.io/user-guide/#using-broccoli-funnel).

**Comment by leo** [comment]
@FelixAkk Your suggestions are now live, btw: http://ember-cli.com :blush:

**Comment by FelixAkk** [comment]
Thanks a lot for taking up the feedback, and implementing it so quickly! :)

**Comment by stefanpenner** [comment]
:+1:
